Dockerfile

```diff
@@ -7,9 +7,9 @@ COPY environment.yml /
 RUN conda env create --quiet -f /environment.yml && conda clean -a
 
 # Add conda installation dir to PATH (instead of doing 'conda activate')
-ENV PATH /opt/conda/envs/nf-core-cageseq-1.0.0/bin:$PATH
+ENV PATH /opt/conda/envs/nf-core-cageseq-2.0.0dev/bin:$PATH
 # Dump the details of the installed packages to a file for posterity
-RUN conda env export --name nf-core-cageseq-1.0.0 > nf-core-cageseq-1.0.0.yml
+RUN conda env export --name nf-core-cageseq-2.0.0dev > nf-core-cageseq-2.0.0dev.yml
 
 # Instruct R processes to use these empty files instead of clashing with a local version
 RUN touch .Rprofile
```

Comment:
Could we not use biocontainers for all our things and with that get rid of this Dockerfile and environment.yml?

Reply:
Yeah had them only in there because I wasn't sure how we'll go ahead with that ....
Simply deleted them now.